carrot_cli/src/carrot_cli/software/command.py

```diff
@@ -34,6 +34,12 @@ def find_by_id(id):
     default="",
     help="The url of the repository where the software code is hosted",
 )
+@click.option(
+    "--machine_type",
+    default="",
+    help="Optional machine type for Google Cloud Build to use for building this software.",
+    type=click.Choice(["standard", "n1-highcpu-8", "n1-highcpu-32", ''], case_sensitive=False)
```

Comment:
I would probably add the actual machine type for standard as well if that's possible so people can choose between "Give me the default" and "Give me this specific small machine" in case the default changes in the future.

Reply:
That makes sense.  I can make that change

Reply:
bizarre, lets leave it as is then